bugbear.py

```diff
@@ -518,6 +518,10 @@ def visit_Import(self, node):
         self.check_for_b005(node)
         self.generic_visit(node)
 
+    def visit_Set(self, node):
```

Comment:
Can we do this for dict keys too? It should be fairly easy and it's also a reasonably common bug.

Reply:
Pyflakes (which is included in flake8) has a rule to detect duplicate keys with **different** values, [F601 dictionary key name repeated with different values](https://flake8.pycqa.org/en/latest/user/error-codes.html).

It doesn't flag duplicate keys with the same values, but as that's unlikely to cause a bug per se, should we check for that?

Reply:
Thanks for the extra context! Yes I think dicts with both keys and values the same are pretty much equivalent to duplicate set members. Honestly I don't have a great sense of what rules belong where, but maybe to avoid duplication we should only alert in cases where pyflakes doesn't?

Reply:
I agree, we would only want to flag when both the dictionary keys and values are the same to avoid conflicts with Pyflakes. 

I think a new rule would be good for this (in case people want to disable each rule separately) and we could leave B033 as it is? As we'd be checking both the keys and the values it also wouldn't easily fit into this rule anyway, so I don't think we'd gain much by combining them into a single rule.